### ISSUE TITLE
What happened?

### ISSUE THREAD

**Issue by ghost** [cataclysmbnteam/Cataclysm-BN#29]
Everything was looking so promising and then it seems like everyone involved with this just disappeared. I can only imagine hoods were thrown over you and you were all taken into unmarked vans.

**Comment by Coolthulhu** [comment]
My job eats most of my time nowadays and I'm not very good at managing the rest of it.
I didn't see a lot of interest last time (though maybe I looked in wrong places), I don't know if anyone would actually play it if I updated it.

**Comment by Aidolii** [comment]
Welp, that sucks. It incorporated a lot of common sense tweaks that, in my opinion, made the game much more playable.

I'm honestly close to giving up on Cataclysm as a whole, it seems like the project is stuck in development hell. I normally don't advocate for feature freezes, but good god.

**Comment by AlecWhite** [comment]
I'm still playing now and then a version from last October.  If you don't update it, I'm afraid I'll forget about the game and stop playing it.
On a side note, this motivated me keep up with small /prog/ projects, kudos for that Cool, and thanks for all your previous work.

**Comment by Aidolii** [comment]
Honestly, if you've really lost most of your interest in the project, I would at least love it if you could have a fork more reduced in scope that just fixes the most **egregious** problems in the current build of CDDA. I'm not up to date on how the current build plays, but I remember how aspects of aiming were essentially broken for months at a time, and how it was impossible to hit enemies early on without throwing objects at them.

I know right now throwing is still overpowered and Kevin keeps turning down all possible solutions to the problem.

Basically if you could just make a version of the game that implements no-brainer changes or stopgap solutions that don't require much maintenance on your end (so you can merge most PRs without a second thought) that'd be swell.

If that means making your fixes not quite as good as they were before for the sake of being easier to update, I mean, let's face it, it's better than nothing!

**Comment by rtizzy** [comment]
@Aidolii 

Hey. Sorry dude. @Coolthulhu is the primary developer and I was mostly around to help with CI/CD, website, and organization. 

If you're interested or able to send a few devs our way we might be able to get something going. 

It's actually a big FOSS event right now 

https://hacktoberfest.digitalocean.com/

So if there was ever a time to try and build momentum, now would be it. 

@Coolthulhu 

Let me know if you're down to get started again sometime soon.

**Comment by Coolthulhu** [comment]
I'm back for now.
